File: personnal-backend/personnal-backend/src/main/java/com/example/personnalbackend/Entity/Personnal.java
package com.example.personnalbackend.Entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Personnal {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;
    private String fullName;
    private String address;
    private String phoneNumber;
}

File: flutter_personnal/lib/models/personnal.dart
class Personnal {
  final int id;
  final String fullName;
  final String address;
  final String phoneNumber;

  Personnal({
    required this.id,
    required this.fullName,
    required this.address,
    required this.phoneNumber,
  });

  factory Personnal.fromMap(Map personnalMap) {
    return Personnal(
      id: personnalMap['id'],
      fullName: personnalMap['fullName'],
      address: personnalMap['address'],
      phoneNumber: personnalMap['phoneNumber'],
    );
  }
}
